### PR TITLE
Dev run script and --dir console argument

### PR DESCRIPTION
**Pull request by immackay**
Optimized argument parsing

--dir<path/to/library/folder> argument to set path for library imports

Included shell and batch scripts to easily run Qupla from the root of the git repo.

Ignored `.gradle` and `.DS_Store` in .gitignore for local Gradle distributions and Mac OS directory modifications.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ qupla.iws
 out
 .idea
 build
+.gradle
+.DS_Store
```
Comment (jserv) [inline .gitignore]: This change is not related to the subject you managed to propose.

Comment (immackay) [reply .gitignore]: Sure. But many developers use both a local gradle distribution, and Mac OS. Changing the .gitignore is fairly common, and having a proper .gitignore for all users is standard practice on public repositories. For another example, see the changes that were approved in PR #23, or read more about gitignore files at https://git-scm.com/docs/gitignore. There are also some standard .gitignore files listed [here](https://github.com/github/gitignore). 

Specifically, please look at the top of these files: [https://github.com/github/gitignore/blob/master/gradle.gitignore](gradle.gitignore) and [https://github.com/github/gitignore/blob/master/Global/macOS.gitignore](macOS.gitignore)
